### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by mpa-mxiang**
For this PR, I:

- Use webpack to bundle JavaScript.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+];
+
+function renderTasks() {
+  const todoList = document.getElementById('todo-list');
+
+  tasks.sort((a, b) => a.index - b.index);
+
+  tasks.forEach((task) => {
+    const taskItem = document.createElement('li');
+    taskItem.innerHTML += `<div class="square"></div> <p>${task.description}</p> <i class="fa">&#xf142;</i>`;
```
Comment (AdnanOlarmmi) [inline src/index.js]: - [x] Nice one dynamically rendering the tasks dynamically, but kindly note that the `checkbox` or in your case the  `<div class="square"></div>` should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @mpa-mxiang, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Tasks dynamically listed.
- Webpack properly setup.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (mpa-mxiang) [reply src/index.js]: Thank you for your example!

**Review by DuaneDave — APPROVED** [review]
Hi @mpa-mxiang 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ DOM elements are rendered correctly
✔️ Nicely looking UI

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mpa-mxiang** [comment]
Thank you for the approval!

**Pull request merged**